tests/lib/extractors/test_review_model.py

```diff
@@ -0,0 +1,167 @@
+"""Tests for the text metadata extractors."""
+import unittest
+from unittest.mock import patch, MagicMock
+
+import numpy as np
+import pandas as pd
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import MultiLabelBinarizer
+
+from lib.extractors.review_model import (
+    ReviewModel, TrainingData, SingleValue
+)
+
+
+class TestReviewModel(unittest.TestCase):
```

Comment:
we use `pytest` for unit testing, rather than `unittest`. would it be possible to refactor this functionality to follow `pytest` patterns?